File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

# Loop till getting the proper values
while True:
    height = get_int("Height: ")
    if 0 < height < 9:
        break

for i in range(height):
    # Print blanks
    for j in range(height - (i + 1)):
        print(" ", end='')
    # Print blocks
    for k in range(i + 1):
        print("#", end='')
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        //高さを受け取る
        height = get_int("height: ");

    }
    while (height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height - (i + 1); j++)
        {
            //空白を入力
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++)
        {
            //#を入力
            printf("#");
        }
        printf("\n");
    }
}